src/control/system/database_test.go

```diff
@@ -384,3 +384,183 @@ func TestSystem_Database_BadApply(t *testing.T) {
 		})
 	}
 }
+
+func raftUpdateTestMember(t *testing.T, db *Database, op raftOp, member *Member) {
+	mu := &memberUpdate{
```

Comment:
Best to call t.Helper() so that a failure in this function is more accurately associated with the test that called the helper.

Reply:
Oops, good catch. Fixed.